roles/calico/meta/main.yml

```diff
@@ -15,3 +15,4 @@ galaxy_info:
 dependencies:
 - role: openshift_facts
 - role: container_runtime
+- role: kube_proxy
```

Comment:
In the release-3.10 branch, this role dependency should go in the `calico_master` role. If this is still a thing going forward, we should leave this in the calico role (we will rename the roles going forward as per https://github.com/openshift/openshift-ansible/pull/9435). I made a PR for the appropriate calico fixes in the `release-3.10` branch here: https://github.com/openshift/openshift-ansible/pull/9657 .

Reply:
exactly what @mgleung , for the 3.10 release branch this should be under `calico_master` role.